gallery/src/components/NxToggle/NxToggleGaplessExample.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useState } from 'react';
+
+import { NxToggle } from '@sonatype/react-shared-components';
+
+function NxToggleExample() {
+  // this example uses the `useState` hook for succinctness, but you could also manage the state manually
+  // in a class component
```

Comment:
This comment is not something that needs to be duplicated in new copy/pasted examples.  In fact if anything I think the react community has moved far enough beyond class components that I'd rather not have this comment anywhere

Reply:
> This comment is not something that needs to be duplicated in new copy/pasted examples. In fact if anything I think the react community has moved far enough beyond class components that I'd rather not have this comment anywhere

Removed here should we have a ticket for removing these comment more broadly? 

https://github.com/sonatype/sonatype-react-shared-components/pull/309/commits/3885453a58bba05042c563730145acc81412887b